homeassistant/components/switchbee/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "SwitchBee",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/switchbee",
-  "requirements": ["pyswitchbee==1.5.5"],
+  "requirements": ["pyswitchbee==1.6.1"],
```

Comment:
I don't see 1.6.1 released on github yet?

Reply:
I updated the PR and the description 

on 1.6.1 I an old typing issue that just appeared in Mypy and also added license.